ingress/vendor/k8s.io/kubernetes/pkg/client/unversioned/testclient/fake_componentstatuses.go

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
```

Comment:
Is this all from the main repo generated code? do we need all of it or can we do with something simpler?